README.md

```diff
@@ -1,6 +1,25 @@
 # twitter-personal-analyzer-bot
 Analyze your interactions.
 
+## Set up
+In `src\main\resources\google-credentials.json`, create the following file :
```

Comment:
This does not make sense, your users are most likely not going to build your project, hence resources cannot be available and read from this folder.
Why not read from the root folder, provide an argument to the CLI command or create a CLI method to init a file from where you'll read data?

Reply:
I just had in mind the same way that the TwitterClient is using the `twitter-credentials.json` data.
But i totally agree, the best solution should be to provide some argument, but as far as I need like 10 parameters to access the Google Sheets API, does it make sense to have so many args ?